tools/cli/commands/generate.js

```diff
@@ -377,11 +377,18 @@ async function generatePluginFromStarter( projDir, answers ) {
 	} );
 	files = files.split( '\n' ).map( str => str.replace( 'projects/plugins/starter-plugin', '' ) );
 	files.forEach( file => {
-		if ( file ) {
+		if ( file && ! file.startsWith( 'changelog/' ) ) {
```

Comment:
Is there a real-world case where `file` wouldn't exist here?

Reply:
I think every run will have an empty-string entry at the end of the array.